examples/sps/multiapp/complete_geometry/electrothermal/engineering_scale_electrothermal.i

```diff
@@ -620,10 +620,8 @@ initial_temperature=293 #roughly 600C where the pyrometer kicks in
     gap_conductivity_function_variable = temperature
     normal_smoothing_distance = 0.1
   []
-[]
 
 ##Thermal Contact between gapped graphite die components
-[ThermalContact]
```

Comment:
@sapitts Do you have context as to why multiple `ThermalContact` blocks were added in this file? Purely for organization, or was this for another reason related to the action?

Reply:
I can't find anything in the github history. I'd guess that separate blocks were used to indicate components in contact at the start of the simulation (vertically aligned) vs components that have a gap in the starting mesh geometry to account for the graphite foil. Since the comment is retained in this PR, I'm fine with this change